app/containers/GamePage/bang.js

```diff
@@ -6,10 +6,9 @@ export default class Bang extends Component {
   static propTypes = {
     assign: PropTypes.bool,
     database: PropTypes.object,
-    email: PropTypes.string,
-    params: PropTypes.object,
+    emailKey: PropTypes.string,
     participants: PropTypes.array,
-    players: PropTypes.number,
+    rid: PropTypes.string,
```

Comment:
Though I'm fine if you leave it rid for now but in the future you should label things so its easy for new people looking at code to figure out what some variables mean